### PR TITLE
Cockpit 135, 136 and 137

### PR DESCRIPTION
**Pull request by dperpeet**
Things I left out:
- It is now possible to use file descriptors passed over the DBus API
- Make privilege escalation work with sudo reauthorization

What do you think @stefwalter ?
Left to do:
- [x] Video for Locked/Unlocked indicator
- [x] Better text for ssh connections via cockpit-bridge instead of cockpit-ws

### CODE REVIEW

File: content/cockpit-135.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 135.
+
+### Redesigned sidebar navigation
+
+The existing page menu took some getting used to for some use cases. It's now easier to use when connected to multiple
+hosts and provides the basis future improvements that will reduce or remove the horizontal menu entries. On systems
```
Comment (martinpitt) [inline content/cockpit-135.md]: basis "for"

File: content/cockpit-135.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+the user drop down menu, named ```Active Pages```. It only becomes visible when the ```ALT``` key is pressed while
+clicking on the menu dropdown. On some drag enabled browsers it doesn't work to just use ```ALT```, but any combination
+involving ```ALT```, such as ```CTRL+ALT```, also works. Check out the screenshot below for a peek.
+
+![Virtual machine disks](chttp://cockpit-project.org/blog/images/cockpit-active-pages.png)
+
+### SSH connections established within the user session
+
+When one Cockpit instance connects to other machines it does so via SSH. Previously
+these connections were launched from cockpit-ws the process listening on the network.
```
Comment (martinpitt) [inline content/cockpit-135.md]: Comma after -ws, or parentheses.

File: content/cockpit-135.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+As part of making Cockpit mirror standard Linux practices better, SSH connections are
+now made from within the logged in user session, in launched from the cockpit-bridge
```
Comment (martinpitt) [inline content/cockpit-135.md]: Drop the "in"? (not entirely sure what you wanted to say, but without that it makes sense)

File: content/cockpit-135.md (hunk 1 above)
Comment (dperpeet) [inline content/cockpit-135.md]: added

File: content/cockpit-135.md (hunk 2 above)
Comment (dperpeet) [inline content/cockpit-135.md]: added

File: content/cockpit-135.md (hunk 3 above)
Comment (dperpeet) [inline content/cockpit-135.md]: done

File: content/cockpit-137.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+Check out the screenshot below to see how this looks now.
+
+![Sidebar order](chttp://cockpit-project.org/blog/images/cockpit-menu-order.png)
+
+### Storage management is more convenient
+
+In order to make configuring storage more convenient, the Cockpit UI now prevents removing disks from a RAID when
+removal would lead to data loss due to an insufficient number of remaining volumes in the RAID. Also, when creating
+a Volume Group (LVM) or adding disk space to a Volume Group, unpartitioned space is now also offered as a choice.
```
Comment (martinpitt) [inline content/cockpit-137.md]: Nitpick: duplicate "also"

File: content/cockpit-137.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+ * System information
+ * Logs for troubleshooting
+ * Configuring major subsystems: Storage, Networking
+ * What's running: Containers and Virtual Machines
+ * Implementation details: Admin accounts, Services/Units
+
+Check out the screenshot below to see how this looks now.
+
+![Sidebar order](chttp://cockpit-project.org/blog/images/cockpit-menu-order.png)
```
Comment (martinpitt) [inline content/cockpit-137.md]: What's the magic behind these `chttp://` links? I first thought it was a typo, but it seems that all links in previous commit have that too.

File: content/cockpit-137.md (hunk 4 above)
Comment (dperpeet) [inline content/cockpit-137.md]: removed

File: content/cockpit-137.md (hunk 5 above)
Comment (dperpeet) [inline content/cockpit-137.md]: Nicest way ever of pointing out my copy & paste errors :)

**Comment by dperpeet** [comment]
Updated, thanks for catching these errors.

**Comment by martinpitt** [comment]
Cheers!

**Pull request merged**